README.md

```diff
@@ -1,2 +1,99 @@
-# mouse-presence
-Centralizes handling of mouse appearance/disappearance events in the browser
+# Mouse Presence
+
+Centralizes handling of mouse appearance/disappearance events in the browser.
+
+
+# Installation
+
+npm: `npm install --save mouse-presence`
+
+Yarn: `yarn add mouse-presence`
+
+
+# API Reference
+
+## `MousePresence.onDisappear(cb)`
+
+`cb`: A callback function that takes no arguments and returns nothing.
+
+returns: A reference to `cb`.
+
+Registers a callback to be called whenever a mouse cursor disappearance is dispatched.
+
+## `MousePresence.onAppear(cb)`
+
+`cb`: A callback function that takes no arguments and returns nothing.
+
+returns: A reference to `cb`.
+
+Registers a callback to be called whenever the mouse is moved after a disappearance was dispatched.
+(Moving the mouse after the cursor disappeared will cause it to reappear.)
+
+## `MousePresence.unregisterCallback(cb)`
+
+`cb`: The callback reference to be unregistered.
+
+returns: Nothing.
+
+Unregisters a previously-registered callback from either onDisappear or onAppear (or both).
+If no such callback is found, nothing happens.
+
+## `MousePresence.dispatchDisappear()`
+
+returns: Nothing.
+
+Dispatches a mouse disappearance event, which will call all registered onDisappear callbacks (will
+only do this once - subsequent calls will have no effect until the next onAppear event occurs).
+It will also register a one-time mousemove listener on the document, which when triggered, will
+call all registered onAppear callbacks.
+
+## `MousePresence.isPresentNow()`
+
+returns: Boolean indicating whether the mouse cursor is present.
+
+Determine if the mouse cursor is present or not (according to the last reported
+appearance/disappearance event).
+
+
+# Examples
+
+Component A:
+
+```javascript
+let MousePresence = require("mouse-presence");
```

Comment:
nit: maybe the example could use `import` syntax. The `require` syntax is non standard.

Reply:
Good idea, done